### PR TITLE
Params with enum and default

### PR DESCRIPTION
**Pull request by pksunkara**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "fury-adapter-swagger",
-  "version": "0.8.0-pre.10",
+  "version": "0.8.0-pre.12",
```
Comment (honzajavorek) [inline package.json]: why jumping 2 versions in one PR?

File: src/parser.js [hunk 2]
```diff
@@ -798,25 +798,83 @@ export default class Parser {
...
+
+    if (parameter.enum) {
       element = new ArrayElement();
+      element.element = 'enum';
 
-      if (parameter.items) {
-        element.content = [this.convertParameterToElement(
-          parameter.items, (path || []).concat(['items']), true)];
-      }
+      _.each(parameter.enum, (value, index) => {
```
Comment (honzajavorek) [inline src/parser.js]: if `parameter.enum` is array, then you can use built-in `parameter.enum.forEach((value, index) => {`

Comment (pksunkara) [inline src/parser.js]: I know. But we consistently use `_.each` in this repo.

File: package.json (hunk 1 above)
Comment (pksunkara) [inline package.json]: Even is for master branch prereleases and Odd if for beta branch prereleases.

**Pull request merged**